src/library/scala/collection/generic/IsSeqLike.scala

```diff
@@ -11,34 +13,78 @@ package generic
   *
   * @see [[scala.collection.generic.IsIterableLike]]
   */
-trait IsSeqLike[Repr] {
-  /** The type of elements we can traverse over. */
-  type A
-  /** A conversion from the representation type `Repr` to `SeqOps[A, Seq, Repr]`. */
-  val conversion: Repr => SeqOps[A, Seq, Repr]
+trait IsSeqLike[Repr] extends IsIterableLike[Repr] {
+
+  @deprecated("'conversion' is now a method named 'apply'", "2.13.0")
+  override val conversion: Repr => SeqOps[A, Iterable, C] = apply(_)
+
+  /** A conversion from the type `Repr` to `SeqOps[A, Iterable, C]` */
+  def apply(coll: Repr): SeqOps[A, Iterable, C]
 }
 
 object IsSeqLike {
   import scala.language.higherKinds
 
-  implicit val stringRepr: IsSeqLike[String] { type A = Char } =
+  implicit def seqOpsIsIterableLike[CC0[X] <: SeqOps[X, Iterable, CC0[X]], A0]: IsSeqLike[CC0[A0]] { type A = A0; type C = CC0[A0] } =
```

Comment:
should this be named `seqOpsIsSeqLike` ?

Reply:
Yes.